cbrain_task/civet/views/_task_params.html.erb

```diff
@@ -322,7 +322,7 @@
     <th>PD found?</th>
     <th>Mask found?</th>
     <th title="Use T1, T2 and PD native files for tissue classification"><%= select_all_checkbox 'use_multispectral' %>Use multispectral?</th>
-    <th title="Use T1, T2 and PD native files for brain masking"><%= select_all_checkbox 'use_spectral_mask' %>Use mask?</th>
```

Comment:
Is the title correct?

Also why does the messahe say "use spectral_mask" instead of "use spectral mask" ? Messages to users should be in normal text, not programmer text with underscores in them.

Reply:
I verified with Claude last fall.